content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,324 @@
+and supports the same authenticated identity (mutual TLS or JWT), condition and other
+primitives (e.g. IP, port and etc.) as the `v1alpha1` policy.
+
+See [authorization concept page](/docs/concepts/security/#authorization) for a detailed
+in-depth explanation of the `AuthorizationPolicy`.
+
+## Future of the `v1alpha1` policy
+
+The `v1alpha1` RBAC policy (`ClusterRbacConfig`, `ServiceRole`, and `ServiceRoleBinding`) is
+deprecated by the `v1beta1` authorization policy and will not be supported from Istio 1.6 onwards.
+
+Istio 1.4 and 1.5 will continue to support the `v1alpha1` RBAC policy to give you enough
+time to move away from the alpha policies.
+
+## Migration to the `v1beta1` policy
+
+Before `v1alpha1` policy is fully removed, Istio will support both `v1alpha` and `v1beta1`
+policies at the same time. However, the `v1beta1` policy overrides the `v1alpha1` policy if
+they are both applied to the same workload:
+
+1. If any `v1beta1` policies apply to the workload, the `v1beta1` policy is used and
+   the `v1alpha1` policies on the same workload will all be ignored
+1. If no `v1beta1` policy applies to the workload, the `v1alpha1` policy is used if
+   there is any
+
+{{< warning >}}
+This means when migrating to the `v1beta1`, you need to make sure the converted
+`v1beta1` policy covers all the `v1alpha1` policies applied to the same workload. The
+`v1alpha1` policies applied for the workload will be ignored after you applied the `v1beta1` policies.
+{{< /warning >}}
+
+It's recommended to migrate the `v1alpha1` policy workload-by-workload to make sure you don't
+forget any `v1alpha1` policies:
+
+1. For a given workload, find out all the `v1alpha1` policies applied to it
+1. Convert all the `v1alpha1` policies to `v1beta1` for the workload
+1. Apply the `v1beta1` policy and monitor the traffic to make sure the new policy is working as expected
+1. Continue the process for the next workload until all `v1alpha1` policies are converted
+1. Now you can remove all your `v1alpha1` policies
+
+To help ease the migration, Istio provides the command `istioctl x authz convert` to automate the
+conversion from `v1alpha1` to `v1beta1`. The command fetches all the `v1alpha1` policies currently
+applied in the mesh and converts them to the `v1beta1` policy. The tool finds out the service-workload
+mapping by looking at the current services applied in the mesh.
+
+The command does the conversion at its best effort and may not be able to handle some corner cases.
+You should always review the converted policies manually before applying them into your cluster.
+
+## Summary
+
+The Istio `v1beta1` authorization policy is a major update to the previous `v1alpha1` RBAC policy
+with the design principle of aligning with Istio configuration model, improving user
+experience by simplifying the API and making policy semantics more flexible without
+introducing much complexity.
```

Comment:
I am not sure about the "more flexible" part. Probably you can directly mention things like "ingress/egress support", and "mesh level policy".

Reply:
I changed it to `supporting more use cases (e.g. Ingress/Egress gateway)`